sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DocumentResult.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The DocumentResult model.
+ */
+@Immutable
+public class DocumentResult {
+    private final String id;
+    private final TextDocumentStatistics textDocumentStatistics;
+    private final Error error;
+    private final boolean isError;
+
+    DocumentResult(final String id, final Error error, final boolean isError) {
```

Comment:
I'm a bit confused at the last parameter `isError`. if `error != null`, then isn't `isError == true`?